runtime/kusama/src/lib.rs

```diff
@@ -909,6 +910,10 @@ impl InstanceFilter<Call> for ProxyType {
 			ProxyType::IdentityJudgement => matches!(c,
 				Call::Identity(pallet_identity::Call::provide_judgement(..)) |
 				Call::Utility(..)
+			),
+			ProxyType::CancelProxy => matches!(c,
+				Call::Proxy(pallet_proxy::Call::remove_announcement(..)) |
+				Call::Proxy(pallet_proxy::Call::reject_announcement(..))
```

Comment:
This can be removed. `reject_announcement` should be called from the account that sets the proxy. We just want to be able to cancel it from the proxy account (which `remove_announcement` does).